src/modules/frost/main_impl.h

```diff
@@ -62,7 +61,7 @@ static void secp256k1_gej_mul_scalar(secp256k1_gej *result, const secp256k1_gej
         return;
     }
     secp256k1_ge_set_gej_safe(&pt_ge, pt);
-    secp256k1_ecmult_const(result, &pt_ge, sc, ECMULT_CONST_256_BIT_SIZE);
+    secp256k1_ecmult_const(result, &pt_ge, sc);
```

Comment:
The removal of `ECMULT_CONST_256_BIT_SIZE` is similar to what upstream has done in:

https://github.com/bancaditalia/secp256k1-frost/pull/24/files#diff-038202ab5b1c3d61a58d26556c798ac0ab5fc636902af1a5dae6fd5298fbcfe4R52-R54